apps/src/templates/levelSummary/MultiResponses.jsx

```diff
@@ -42,82 +33,78 @@ const MultiResponses = ({scriptData, showCorrectAnswer = false}) => {
   ];
 
   const answers = scriptData.level.properties.answers;
-  const answerData = multiAnswerCounts(scriptData.responses, answers.length);
-  const answerMax = Math.max(...Object.values(answerData));
-  const correctAnswers = answers.reduce((acc, cur, i) => {
-    if (cur.correct) {
-      acc.push([...LETTERS][i]);
-    }
-    return acc;
-  }, []);
-
-  // FIXME: Is this too weird?
-  const correctAnswerElement = document.getElementById(
-    'summary-correct-answer'
+  const answerData = useMemo(
+    () => multiAnswerCounts(scriptData.responses, answers.length),
+    [scriptData.responses, answers.length]
+  );
+  const answerMax = useMemo(
+    () => Math.max(...Object.values(answerData)),
+    [answerData]
+  );
+  const correctAnswers = useMemo(
+    () =>
+      answers.reduce((acc, cur, i) => {
+        if (cur.correct) {
+          acc.push([...LETTERS][i]);
+        }
+        return acc;
+      }, []),
+    [answers]
   );
```

Comment:
@bethanyaconnor does this seem right? it worked, at least

Reply:
looks good to me!